### ISSUE TITLE
jsonize dispersion multiplier for Targeting System CBM

### ISSUE THREAD

**Issue by scarf005** [cataclysmbnteam/Cataclysm-BN#2812]
### Discussed in https://github.com/cataclysmbnteam/Cataclysm-BN/discussions/2788

<div type='discussions-op-text'>

<sup>Originally posted by **NobleJake** May 11, 2023</sup>

![image-244](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/108422062/4e276420-0aae-46ea-9383-165b73aac320)

The current ranged weapon dispersion multiplier function of Targeting System CBM is hardcoded, and it would be better if this aspect could be adjusted more flexibly. The reasons for this are as follows:

Firstly, if the dispersion multiplier can be adjusted through items at the time of equipping or wearing, it would be possible to develop items such as tactical sights.

Secondly, if this feature can be applied not only to items but also to effects, it would be possible to create debuffs that greatly reduce the hit rate by increasing the dispersion multiplier of enemy NPCs.

Therefore, if the hardcoded part of Targeting System CBM is Jsonized (e.g. item(not gun mod) dispersion modifier: 0.5), it would be useful in many aspects if this feature could be utilized in various ways.

![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/108422062/dbd23461-b4a7-4259-9e44-e2ce376a17b3)

In simple terms, the sentence means "To put it easily, it suggests expanding the dispersion_modifier in gun mods to clothing or effects."</div>
